packages/asset-swapper/src/utils/quote_report_generator.ts

```diff
@@ -116,6 +137,85 @@ export function generateQuoteReport(
     };
 }
 
+/**
+ * Generates a report of sources considered while computing the optimized swap quote, the sources ultimately included in the computed quote, and all considered quotes.
```

Comment:
can you specify in this comment how this method differs from `generateQuoteReport`?